pkg/deploy/che_configmap.go

```diff
@@ -128,6 +128,9 @@ func GetConfigMapData(cr *orgv1.CheCluster) (cheEnv map[string]string) {
 
 	ingressDomain := cr.Spec.K8s.IngressDomain
 	tlsSecretName := cr.Spec.K8s.TlsSecretName
+	if tlsSupport && tlsSecretName == "" {
```

Comment:
`len(tlsSecretName) < 1`

Reply:
What is the benefit of `len(tlsSecretName) < 1` over `tlsSecretName == ""`?

Reply:
no benefits)